core-bundle/src/Twig/Inheritance/TemplateHierarchyInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Twig\Inheritance;
+
+interface TemplateHierarchyInterface
```

Comment:
We technically do not need an interface for this. But it felt wrong to reference the `ContaoFilesystemLoader` (which currently implements the functionality), thus decoupling it.